### ISSUE TITLE
Training Setting

### ISSUE THREAD

**Issue by jiaweihhuang** [THUKey/pytorch-mobilenet-v3#4]
Hi, could you share your training setting for Large or Small models? 

Thanks!

**Comment by cxxgtxy** [comment]
+1 Thanks.

**Comment by kuan-wang** [comment]
number of epochs: 150
learning rate schedule: cosine learning rate, initial lr=0.05
weight decay: 4e-5
remove dropout

**Comment by cxxgtxy** [comment]
How about the batch size, thanks

**Comment by kuan-wang** [comment]
batch size: 256

**Comment by cxxgtxy** [comment]
Thanks. I use the above hypers, cannot reproduce, only 66.7%.
